nemo/collections/asr/models/confidence_ensemble.py

```diff
@@ -135,7 +136,10 @@ def compute_confidence(hypothesis: Hypothesis, confidence_cfg: ConfidenceConfig)
         alpha = confidence_cfg.method_cfg.temperature
     conf_func = get_confidence_measure_bank()[conf_type]
 
-    conf_value = aggr_func(conf_func(filtered_logprobs, v=vocab_size, t=alpha)).cpu().item()
+    if filtered_logprobs.shape[0] == 0:  # for the edge-case of all blanks, no frames
```

Comment:
I'm not sure that returning 0 is the best choice here. The model might be very confident in predicting only blanks when there is a silence and we probably shouldn't ignore that. So maybe safer to use the same approach as for rnnt and return a confidence of a single blank token. Something like this in the `get_filtered_logprobs` function
```
            if filtered_logprobs.shape[0] == 0:  # for the edge-case of all blanks
                filtered_logprobs = logprobs[:1]
```

Reply:
Good point! Fixed this issue in the latest commit by following a similar approach for CTC as done for RNNT in get_filtered_logprobs(). 
Still investigating the rnnt normalization issue of why log_normalize isn't being propagated as true